Decsys/ClientApp/src/app/components/ComponentRender/ComponentRender.js

```diff
@@ -1,6 +1,6 @@
 import React, { cloneElement } from "react";
 
-const ComponentRender = ({ component, params }) => {
+const ComponentRender = ({ component, actions, params }) => {
```

Comment:
make the actions parameter consistent in this file with regards to the order in which it is given.